Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -237,6 +237,15 @@ input[type=submit], input[type=button], input[type=reset], input[type=checkbox],
     cursor: unset;
 }
 
+input[type=submit], input[type=button], input[type=reset] {
+    padding: 4px 8px;
+    background-color: -libweb-palette-button;
+    border: 1px solid -libweb-palette-inactive-window-border1;
```

Comment:
The inactive window border color is unrelated to buttons. While we can't specify both threed-highlight and threed-shadow, we can at least use one of the two. :)